libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1231,12 +1234,18 @@ void CompilerUtils::convertType(
 			case DataLocation::Memory:
 				// nothing to do
 				break;
+			case DataLocation::Code:
+				solUnimplemented("\"code\" keyword as a data location is not yet implemented");
+				break;
 			}
 			break;
 		case DataLocation::CallData:
 			solAssert(_typeOnStack == _targetType);
 			// nothing to do
 			break;
+		case DataLocation::Code:
+			solUnimplemented("\"code\" keyword as a data location is not yet implemented");
```

Comment:
I think it might be better to just use `'code' as data location not yet implemented.`